docker/service/src/main/java/uk/nhs/adaptors/scr/controllers/fhir/FhirController.java

```diff
@@ -38,15 +41,23 @@
         produces = {APPLICATION_FHIR_JSON_VALUE, APPLICATION_FHIR_XML_VALUE})
     @ResponseStatus(HttpStatus.OK)
     public WebAsyncTask<ResponseEntity<?>> acceptFhir(
-        @RequestHeader("Content-Type") MediaType contentType, @RequestBody String body)
-        throws FhirValidationException {
+        @RequestHeader("Content-Type") @NotNull MediaType contentType,
+        @RequestHeader("Nhsd-Asid") @NotNull String nhsdAsid,
+        @RequestHeader("Party-Id-From") @NotNull String partyIdFrom,
```

Comment:
I checked apigee setup - partyId is not passed as a header. Let's move it to application.yml for now - until we sort it out.